test/operator-framework-e2e/setup.sh

```diff
@@ -0,0 +1,193 @@
+#! /bin/bash
+
+set -e
```

Comment:
```sh
set -o errexit
set -o nounset
set -o pipefail
```

a) you always want nounset, and pipefails
b) write for reading, not for brevity

Reply:
https://github.com/operator-framework/operator-controller/pull/503/commits/cc277fcd2703a5aaab68e7802480fc8b6fc7412b